releases/release-1.14/README.md

```diff
@@ -0,0 +1,146 @@
+# Kubernetes 1.14
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.14/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.14/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s114-minutes) (join [kubernetes-milestone-burndown@] to receive meeting invites)
+* Contact: [#sig-release] on slack, [kubernetes-sig-release@] on e-mail
+
+#### Tracking docs
+
+* TODO: [Enhancements Tracking Sheet](http://bit.ly/k8s114-enhancements)
+* TODO: [Bug Triage Tracking Sheet](http://bit.ly/k8s114-bugtriage)
+* TODO: [CI Signal Report](http://bit.ly/k8s114-cisignal)
+* [Retrospective Document](http://bit.ly/k8s114-retro)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/sig-release/ephemera/testing.md) (TODO: this guide should live elsewhere, ref: [kubernetes/sig-release#428](https://github.com/kubernetes/sig-release/issues/428))
+
+## tl;dr
+
+The 1.14 release cycle is proposed as follows:
+
+- **Monday, January 07**: Week 1 - Release cycle begins
+- **Tuesday, January 29**: Week 4 - [Enhancements Freeze]
+- **Thursday, March 07**: Week 9* - [Code Freeze]
```

Comment:
RFC: this matches the (historically shortest) 1.5 week code freeze used by 1.13

There is an asterisk because previous releases have followed a pattern of [conditionally](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.12#conditionally-shortened-code-freeze) [pushing](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.11#conditionally-shortening-code-freeze) code freeze back if CI signal looks good.  We could do the same here if people are so inclined.